src/mapToJson.h

```diff
@@ -1,6 +1,11 @@
 #include "OP2Utility.h"
+#if _WIN32
+#include <nlohmann/json.hpp>
+#else
 #include <json.hpp>
+#endif
```

Comment:
Ahh yes, looks like that's the natural include path for the NuGet package. This should probably be fixed on the Linux side by adjusting the include folder option. Looks like I should strip off the "nlohmann/" part of the `-I` option:
```
CPPFLAGS := -I $(UTILITYDIR)include -I packages/nlohmann.json.3.2.0/build/native/include/
```

That should allow it to build on both platforms without the `#if` include guard.